sys/mips/include/pcb.h

```diff
@@ -142,15 +142,15 @@
  * contains the bounds). This means we need to update the offset of PCC to
  * value of PC - PCC.base before writing to EPCC.
  */
-#define RESTORE_U_PCB_PC(pc_vaddr_tmpreg, tmpreg2, pcb)			\
-	/* EPCC is no longer a GPR so load it into KSCRATCH first */	\
-	RESTORE_U_PCB_CREG(CHERI_REG_KSCRATCH, PCC, pcb);		\
-	RESTORE_U_PCB_REG(pc_vaddr_tmpreg, PC, pcb);			\
-	RESTORE_EPCC(CHERI_REG_KSCRATCH, pc_vaddr_tmpreg, tmpreg2);	\
+#define RESTORE_U_PCB_PC(unused_pc_vaddr_tmpreg, pcb)			\
+	/* EPCC is no longer a GPR so load it into C27 first. */	\
+	RESTORE_U_PCB_CREG(CHERI_REG_C27, PCC, pcb);			\
```

Comment:
I think we try to use KSCRATCH instead of C27 in most places?

Reply:
Ah yes good catch. I originally restored C27 after clobbering it, but I *think* I changed the code to ensure that C27 is restored after.
However, this is quite fragile so I should probably use the scratch hwr instead.

Reply:
KSCRATCH is C27, it's just an alias that we use for it IIRC.  I think using the KSCRATCH name would be a nop in terms of functions, just more consistent for humans reading the code.